common/protocols/src/main/proto/app/coronawarn/server/common/protocols/internal/v2/dgc_parameters.proto

```diff
@@ -8,9 +8,19 @@ message DGCParameters {
 
   DGCTestCertificateParameters testCertificateParameters = 1;
   uint32 expirationThresholdInDays = 2;
+  DGCBlocklistParameters blockListParameters = 3;
 }
 
 message DGCTestCertificateParameters {
   uint32 waitAfterPublicKeyRegistrationInSeconds = 1;
   uint32 waitForRetryInSeconds = 2;
-}
\ No newline at end of file
+}
+
+message DGCBlocklistParameters {
+  repeated DGCBlockedUVCIChunk blockedUvciChunks = 1;
+}
+
+message DGCBlockedUVCIChunk {
+  repeated uint32 incides = 1;
```

Comment:
@ioangut  can you update this to the version from https://github.com/corona-warn-app/cwa-server/pull/1595/files? It fixes a typo here: incides -> indices